ai/tools/firecrawl.tsx

```diff
@@ -0,0 +1,52 @@
+import { CurrentWebLoading, CurrentWeb } from "@/components/prebuilt/web";
+import { createRunnableUI } from "@/utils/server";
+import { DynamicStructuredTool } from "@langchain/core/tools";
+import FirecrawlApp from "@mendable/firecrawl-js";
```

Comment:
Would rather use the `FireCrawlLoader` from langchain, I can push the update.

Reply:
Ok awesome!